src/test/resources/testResult/testOneNumberedSeriesPub.xml

```diff
@@ -24,6 +24,9 @@
                 <publication_date media_type="online">
                     <year>2013</year>
                 </publication_date>
+                <publisher_item xmlns="http://www.crossref.org/schema/4.4.2">
```

Comment:
Why is there an extra namespace declaration here?  It should be at the top of the document

Reply:
good catch, taken out.